internal/uvm/create_wcow.go

```diff
@@ -113,6 +114,27 @@ func CreateWCOW(ctx context.Context, opts *OptionsWCOW) (_ *UtilityVM, err error
 		}
 	}
 
+	virtualSMB := &hcsschema.VirtualSmb{
+		DirectFileMappingInMB: 1024, // Sensible default, but could be a tuning parameter somewhere
+		Shares: []hcsschema.VirtualSmbShare{
+			{
+				Name: "os",
+				Path: filepath.Join(uvmFolder, `UtilityVM\Files`),
+				Options: &hcsschema.VirtualSmbShareOptions{
+					ReadOnly:            true,
+					PseudoOplocks:       true,
+					TakeBackupPrivilege: true,
+					CacheIo:             true,
+					ShareRead:           true,
+				},
+			},
+		},
+	}
+
+	if uvm.devicesPhysicallyBacked {
```

Comment:
Couldnt the NoDirectMap option above just be assigned to uvm.devicesPhysicallyBacked? 
```go 
virtualSMB := &hcsschema.VirtualSmb{
		DirectFileMappingInMB: 1024, // Sensible default, but could be a tuning parameter somewhere
		Shares: []hcsschema.VirtualSmbShare{
			{
				Name: "os",
				Path: filepath.Join(uvmFolder, `UtilityVM\Files`),
				Options: &hcsschema.VirtualSmbShareOptions{
					ReadOnly:            true,
					PseudoOplocks:       true,
					TakeBackupPrivilege: true,
					CacheIo:             true,
					ShareRead:           true,
                                        NoDirectmap:       uvm.devicesPhysicallyBacked,
				},
			},
		},
	}
```